sqlmesh/dbt/target.py

```diff
@@ -17,6 +20,11 @@
 from sqlmesh.utils.errors import ConfigError
 from sqlmesh.utils.pydantic import PydanticModel
 
+if sys.version_info >= (3, 9):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
```

Comment:
We use this in a few places. Is it possible to put into a util and include from there?